include/industry_standard/storage.h

```diff
@@ -0,0 +1,50 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2024 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+/**
+ *  Author: Wilfred Mallawa <wilfred.mallawa@wdc.com>
+ *          Alistair Francis <alistair.francis@wdc.com>
+ **/
+
+/** @file
+ * Definitions of SPDM over the Storage as defined in DSP0286
+ **/
+
+#ifndef STORAGE_BINDING_H
```

Comment:
Can we add `spdm_` as prefix for the file name, such as `spdm_storage_binding.h`?